### PR TITLE
Add a new Compilers SIG

### PR DESCRIPTION
**Pull request by AlexandreEichenberger**
Added changes to the repo to introduce a Compilers SIG. If accepted, will create relevant slack channel and investigate a co-leader, if anyone is interested in helping out.

Changes would also have to be done to the steering committee repo, presumably that would be done if this PR is accepted

Signed-off-by: Alexandre Eichenberger <alexe@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The current list of Contributors and Approvers in the SIGs are listed in the [CO
 | Name      | Responsibilities    |
 | ------------------ | ------------- |
 | [Architecture & Infra](infra) | Defining and maintaining the core ONNX format, the build and CI/CD systems for ONNX repositories, publishing release packages for ONNX, the onnx-docker repository, and creating tools to help integrate with and test against the ONNX standard. This SIG is also the defacto owner of files in the main ONNX repository unless explicitly owned by another SIG. |
+| [Compilers](compilers) | Developing and maintaining core compiler technology optimizing and lowering ONNX format. |
```
Comment (mk-nvidia) [inline README.md:22]: Did you mean "..core compiler technology _for_ optimizing..."

**Review by mk-nvidia — APPROVED** [review]

**Review by andife — APPROVED** [review]

File: compilers/README.md [hunk 2]
```diff
@@ -40,9 +40,9 @@ Please sign up at https://slack.lfai.foundation/ and join [onnx-compilers](https
 
 ## Current projects(s)
 
-* Onnx-mlir: ONNX compiler lowering ONNX representation to other MLIR representations, including LLVM representation for ingestion in the LLVM compiler.
+* Onnx-mlir: lowering ONNX representation to other MLIR representations, including representations for ingestion in the LLVM compiler.
   * Description: [High Level](https://www.onnx.ai/onnx-mlir), [GitHub](https://github.com/onnx/onnx-mlir).
-  * Meeting [notes](https://github.com/onnx/onnx-mlir/wiki/Informal-meeting-agenda-and-notes).
+  * Meeting [agneda and notes](https://github.com/onnx/onnx-mlir/wiki/Informal-meeting-agenda-and-notes).
```
Comment (prasanthpul) [inline compilers/README.md:45]: typo: agenda

File: README.md (hunk 1 above)
Comment (AlexandreEichenberger) [reply README.md:22]: Good suggestion, added

File: compilers/README.md (hunk 2 above)
Comment (AlexandreEichenberger) [reply compilers/README.md:45]: fixed

**Comment by AlexandreEichenberger** [comment]
Groq has been very active in developing onnx-mlir, with the second most PR YTD, including PRs that significantly speeded up our compile time by better handling large constant tensors. Groq is also a startup, which would provide some diversity between big and small companies.  Philip leads their compiler team and is interested in being a co-leader. Let me know if that works for the community at large.

**Comment by AlexandreEichenberger** [comment]
@saurabhtangri let me know if you have any questions, suggestions.

**Review by saurabhtangri — APPROVED** [review]

**Pull request merged**